SDJSBridge/SDWebViewController.m

```diff
@@ -130,7 +140,7 @@ - (void)viewDidDisappear:(BOOL)animated {
     // hide placeholder view after view animates away so that when we pop back
     // we do not see a flicker of the previous page while web view's goBack
     // updates the page
-    self.placeholderView.hidden = YES;
+    self.placeholderView.image = nil;
```

Comment:
I believe this should be in `viewDidDisappear`. Setting nil before the vc disappears might cause the image to disappear before the vc is off screen. Thats how it was done in Zoot https://github.com/TheHolyGrail/Zoot/blob/master/Source/Web%20View/WebViewController.swift#L143-L147